DH_Interface/Classes/DHMapVoteCountMultiColumnList.uc

```diff
@@ -45,24 +45,34 @@ function DrawItem(Canvas Canvas, int i, float X, float Y, float W, float H, bool
         DrawStyle = Style;
     }
 
-    // Split the mapname string, which may be consolitated with other variables
-    Split(VRI.MapList[VRI.MapVoteCount[SortData[i].SortItem].MapIndex].MapName, ";", Parts);
+    // Parse the JSON object
+    MapObject = (new class'JSONParser').ParseObject(VRI.MapList[VRI.MapVoteCount[SortData[i].SortItem].MapIndex].MapName);
+
+    // Set the local MapNameString as it is reused
+    if (MapObject != none && MapObject.Get("MapName").AsString() != "")
+    {
+        MapNameString = MapObject.Get("MapName").AsString();
+    }
+    else
+    {
+        MapNameString = VRI.MapList[VRI.MapVoteCount[SortData[i].SortItem].MapIndex].MapName;
+    }
 
     // Map Name
     GetCellLeftWidth(0, CellLeft, CellWidth);
-    DrawStyle.DrawText(Canvas, MenuState, CellLeft, Y, CellWidth, H, TXTA_Left, class'DHMapList'.static.GetPrettyName(Parts[0]), FontScale);
+    DrawStyle.DrawText(Canvas, MenuState, CellLeft, Y, CellWidth, H, TXTA_Left, class'DHMapList'.static.GetPrettyName(MapNameString), FontScale);
 
     // Vote Count
     GetCellLeftWidth(1, CellLeft, CellWidth);
     DrawStyle.DrawText(Canvas, MenuState, CellLeft, Y, CellWidth, H, TXTA_Left, string(VRI.MapVoteCount[SortData[i].SortItem].VoteCount), FontScale);
 
     // Player Range
-    if (Parts.Length >= 5)
+    if (MapObject != none && MapObject.Get("MinPlayers").AsString() != "" && MapObject.Get("MaxPlayers").AsString() != "")
```

Comment:
Change to ```MapObject.Get("MinPlayers") != none``` instead of `MapObject.Get("MinPlayers").AsString() != "`, same goes for `MaxPlayers`.